### ISSUE TITLE
Transactions are locked

### ISSUE THREAD

**Issue by melahi94** [Vanilla-OS/iso-builder#209]
### Issue Description

I clicked on the button update reboot and update into the software app for applying system updates. Then it seems the Nvidia driver could not be installed correctly, and if I want to force the update using vso, but when I try these way, the following message appears:
"WARNING exec.start
Transactions are locked, another one is already running or a reboot is required".
If I reboot the system nothing happen, also if I try to change to the partition B, is exactly the same story.

### Steps to Reproduce

-Both partitions have the same problem 

### On what version of Vanilla OS this happens?

22.10

**Comment by appleshoops** [comment]
I have the same issue with version 22.10

**Comment by melahi94** [comment]
> I have the same issue with version 22.10

Well, I gave up. I couldn't find any solutions so far. Abroot is locked in both partitions. 

It was nice for testing, but I really need something more stable my working computer. I installed this distro specifically for getting stability, but this problem showed me that is not fully stable yet (after less than one week from the installation), so, I will go back to my old distro and look for another stable inmutable distribution or remain there until Vanilla have further documentation
